app/factories/miner_factory.rb

```diff
@@ -0,0 +1,50 @@
+class MinerFactory
+  load 'ethermine_convertions.rb'
```

Comment:
you don't need to load stuff, rails do it for yoy

Reply:
But if i delete that lines doesnt work, and appear the problem that i send u in slack:

uninitialized constant MinerFactory::MinerSettings

And that for each class.

Reply:
you can try 2 ways
- invoke the class from root `::MinerSettings`
- inherit the nested classes in the module like that `MinerFactory::MinerSettings`
- and the not recommended way, add the path in the application.rb to aoutload the folder